unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/PlacesAndEventsSectionComponentController.cs

```diff
@@ -0,0 +1,19 @@
+public interface IPlacesAndEventsSectionComponentController
```

Comment:
You can use `IDisposable` here: `IPlacesAndEventsSectionComponentController : IDisposable`

Reply:
Correct! Done in https://github.com/decentraland/unity-renderer/pull/1238/commits/758ac9b508e15b6c9518db4a59c59f4338c26b21. Thanks!